dev-requirements.txt

```diff
@@ -0,0 +1,24 @@
+# install latest changes in dbt-core
+# TODO: how to automate switching from develop to version branches?
+# git+https://github.com/dbt-labs/dbt-core.git@1.4.latest#egg=dbt-core&subdirectory=core
+git+https://github.com/dbt-labs/dbt-core.git@1.4.latest#egg=dbt-tests-adapter&subdirectory=tests/adapter
```

Comment:
Note for future: depending on the version number of the next release of dbt-mysql, we'll need to change `1.4.latest` to match that version.